### ISSUE TITLE
Train a tiny neural network.

### ISSUE THREAD

**Issue by carrascomj** [carrascomj/gcn-prot#8]
The training loop is built, we just need to assemble it.

* [x] Write a jupyter notebook that gathers the different functions to train a Graph Convolutional Neural Network for the KrasHras experiment.
* [x] Show that the network is learning.
* [ ] Overfitting.

**Comment by carrascomj** [comment]
Currently, is producing a `NaN`s (on the predctions of the model), [see the notebook](https://github.com/carrascomj/gcn-prot/blob/master/notebooks/01_NaN_tiny_network_Kras.ipynb) . Maybe the normalization of the adjacency matrix is compulsory to do the the `spmm` correctly.
